src/Connection.py

```diff
@@ -0,0 +1,28 @@
+import psycopg2
+
+
+class DAO:
+    def __init__(self):
+        self.conn = psycopg2.connect("dbname=app user=postgres password=1221909128 host=localhost")
```

Comment:
move all credentials to separate `Config.py`. Moreover, it should have logic of an alternative initialization via `system environment` (last one to investigate but not necessary to implementation)

Reply:
Yup, I'll do it asap

Reply:
Done